deepchecks/nlp/checks/model_evaluation/weak_segments_performance.py

```diff
@@ -155,7 +155,7 @@ class PropertySegmentsPerformance(WeakSegmentsAbstractText):
     def __init__(self,
                  properties: Union[Hashable, List[Hashable], None] = None,
                  ignore_properties: Union[Hashable, List[Hashable], None] = None,
-                 n_top_properties: Optional[int] = 10,
+                 n_top_properties: Optional[int] = 15,
```

Comment:
may be slow

Reply:
For now prefer this extra slowness to the warning and missing properties...